Sources/Main.cpp

```diff
@@ -502,7 +504,15 @@ void __fastcall TX584Form::FormCreate(TObject *Sender)
     BuildTree(OpFilter, ResFilter);
     if (ParamCount() > 0) {
         LoadFile(ParamStr(1));
-    }
+	}
+
+    PreviousWindow = SetClipboardViewer(Handle);
```

Comment:
Does this automatically trigger `OnClipboard` handler? If not, we should manually call `OnClipboard` to enable or disable `PasteItem` on startup.